crates/ruff_linter/src/rules/flake8_bugbear/snapshots/ruff_linter__rules__flake8_bugbear__tests__B911_B911.py.snap

```diff
@@ -1,168 +1,168 @@
 ---
 source: crates/ruff_linter/src/rules/flake8_bugbear/mod.rs
 ---
-B911.py:5:1: B911 `itertools.batched()` without an explicit `strict` parameter
+[B911]: `itertools.batched()` without an explicit `strict` parameter
+ --> B911.py:5:1
   |
 4 | # Errors
 5 | batched(range(3), 1)
-  | ^^^^^^^^^^^^^^^^^^^^ B911
+  | ^^^^^^^^^^^^^^^^^^^^ Add an explicit `strict` parameter
 6 | batched("abc", 2)
 7 | batched([i for i in range(42)], some_n)
   |
-  = help: Add an explicit `strict` parameter
 
-B911.py:6:1: B911 `itertools.batched()` without an explicit `strict` parameter
+[B911]: `itertools.batched()` without an explicit `strict` parameter
+ --> B911.py:6:1
   |
 4 | # Errors
 5 | batched(range(3), 1)
 6 | batched("abc", 2)
-  | ^^^^^^^^^^^^^^^^^ B911
+  | ^^^^^^^^^^^^^^^^^ Add an explicit `strict` parameter
 7 | batched([i for i in range(42)], some_n)
 8 | batched((foo for foo in cycle()))
   |
-  = help: Add an explicit `strict` parameter
 
-B911.py:7:1: B911 `itertools.batched()` without an explicit `strict` parameter
+[B911]: `itertools.batched()` without an explicit `strict` parameter
+ --> B911.py:7:1
   |
 5 | batched(range(3), 1)
 6 | batched("abc", 2)
 7 | batched([i for i in range(42)], some_n)
-  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ B911
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Add an explicit `strict` parameter
 8 | batched((foo for foo in cycle()))
 9 | batched(itertools.batched([1, 2, 3], strict=True))
   |
-  = help: Add an explicit `strict` parameter
 
-B911.py:8:1: B911 `itertools.batched()` without an explicit `strict` parameter
+[B911]: `itertools.batched()` without an explicit `strict` parameter
+ --> B911.py:8:1
   |
 6 | batched("abc", 2)
 7 | batched([i for i in range(42)], some_n)
 8 | batched((foo for foo in cycle()))
-  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ B911
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Add an explicit `strict` parameter
 9 | batched(itertools.batched([1, 2, 3], strict=True))
   |
-  = help: Add an explicit `strict` parameter
 
-B911.py:9:1: B911 `itertools.batched()` without an explicit `strict` parameter
+[B911]: `itertools.batched()` without an explicit `strict` parameter
+  --> B911.py:9:1
    |
  7 | batched([i for i in range(42)], some_n)
  8 | batched((foo for foo in cycle()))
  9 | batched(itertools.batched([1, 2, 3], strict=True))
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ B911
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Add an explicit `strict` parameter
```

Comment:
I feel like the primary message looks very good in many cases but not all. 

For example here, I think the `help` text was better. I don't think this is something we need to change in rendering. Instead, we should change how the diagnostic is constructed in the rule. 

The airflow rules are similar. The inline texts are too long.